client/Main.css

```diff
@@ -16,8 +16,8 @@
 }
 
 .player-entry {
-	
-	
+
+
```

Comment:
It removes the bolded out spaces, which are tabspaces and replaces them with newlines